openstack/jar/src/main/java/com/sixsq/slipstream/connector/openstack/OpenStackSystemConfigurationParametersFactory.java

```diff
@@ -36,6 +36,7 @@ protected void initReferenceParameters() throws ValidationException {
 		super.initReferenceParameters();
 		super.putMandatoryEndpoint();
 		super.putMandatoryContextualizationType();
+		super.putMandatoryOrchestratorUsernameAndPassword();
```

Comment:
What about using the same wrapper `putMandatoryOrchestratorUsernameAndPassword()` in `SlipStreamConnector-CloudSigma` as well?